x-pack/winlogbeat/module/security/ingest/security.yml

```
@@ -3442,6 +3458,51 @@ processors:
           splitSidList(ctx.winlog.event_data.SidList, params, ctx);
         }
 
+  - set:
+      field: file.name
+      copy_from: winlog.event_data.RelativeTargetName
+      if: |-
+        ctx.event?.code != null &&
+        ["5140", "5145"].contains(ctx.event.code) &&
+        ctx.winlog?.event_data?.RelativeTargetName != null &&
+        ctx.winlog?.event_data?.RelativeTargetName != ""
```

```suggestion
        ctx.winlog.event_data.RelativeTargetName != ""
```
(already known to be non-null by the time we get here due to the previous null check)